autometa/common/kmers.py

```diff
@@ -429,16 +426,16 @@ def embed(
     ):
```

Comment:
Sorry, can't highlight above line (429/426). Are we not converting `os.stat` to `os.path.getsize`? There is still a `os.stat` here.